pkg/scmprovider/client.go

```diff
@@ -47,6 +47,7 @@ type SCMClient interface {
 	// Functions implemented in organizations.go
 	ListTeams(string) ([]*scm.Team, error)
 	ListTeamMembers(int, string) ([]*scm.TeamMember, error)
+	ListOrgMembers(string) ([]*scm.TeamMember, error)
```

Comment:
Not related, but the one function which sticks our is `DeleteStateComments`. Where is this coming from? I would have loved to see `StaleComments` and `DeleteComments`. But that's besides the point for this PR.

Reply:
https://github.com/jenkins-x/lighthouse/blob/master/pkg/scmprovider/issues.go#L126 fwiw